cartridge/clusterwide-config.lua

```diff
@@ -60,8 +60,27 @@ local digest = require('digest')
 local errno = require('errno')
 local errors = require('errors')
 
+local conf_lib = require('conf')
+local topology_lib = require('topology')
+local argparse = require('cartridge.argparse')
+local vars = require('cartridge.vars').new('cartridge.clusterwide-config')
```

Comment:
Would prefer to avoid creating a new `vars` entry in `clusterwide-config`. See next comment.